### PR TITLE
Adds suggestions to Glossary for capitalization, punctuation, and sense

### PR DESCRIPTION
**Pull request by chrisbobek**

### CODE REVIEW

File: glossary.asciidoc [hunk 1]
```diff
@@ -16,13 +16,13 @@ Comment from Gitter:
 ////
 
 Account::
-    An object containing an address, balance, and nonce, and optional storage and code. An account can be a contract account or an EOA (externally owned account).
+    An object containing an address, balance, nonce, and optional storage and code. An account can be a contract account or an EOA (externally owned account).
 
 Address::
-    Most generally, this represents an EOA or contract, which can receive (destination address) or send (source address) transactions on the blockchain. More specifically, it is the right-most 160 bits of a Keccak hash of an ECDSA public key.
+    Most generally, this represents an EOA or contract thath can receive (destination address) or send (source address) transactions on the blockchain. More specifically, it is the right-most 160 bits of a Keccak hash of an ECDSA public key.
```
Comment (fjrojasgarcia) [inline glossary.asciidoc]: thath -> that?

Comment (chrisbobek) [reply glossary.asciidoc]: 'that'

How embarrassing! I can't believe I missed my own typo. :(

Sorry about that, Javier.

--Chris

Comment (fjrojasgarcia) [reply glossary.asciidoc]: To err is human. :-)
To solve it, you'd need to correct and repush the new commit in the branch of this PR.
In this way, the change will be automatically reflected in this PR.

I take this opportunity to advise the use of branches other than the "develop" branch, trying to keep it clean and synchronized with the upstream.

Comment (chrisbobek) [reply glossary.asciidoc]: Thanks, Javier.
Which branch(es) would you recommend besides "develop"?

**Pull request merged**